tests/acceptance/test_dbus.py

```diff
@@ -109,6 +109,13 @@ def test_dbus_get_jwt_token(self, bitbake_variables, connection, setup_mock_serv
             assert f'string "{self.JWT_TOKEN}' in output
             if version_is_minimum(bitbake_variables, "mender-client", "3.3.1"):
                 assert 'string "http://127.0.0.1:' in output
+                # Check that this really is the only port we're listening on.
+                lines = connection.run("netstat -t -u -l -p -n").stdout.split("\n")
+                lines = [line for line in lines if line.strip().endswith("/mender")]
+                assert len(lines) == 1
+                listen = lines[0].split()[3]
```

Comment:
a bit of a nit, but I always say it: `local_address_column_index=3` (as in: could we have it defined instead of a magic number?)

Reply:
Alright, makes sense. Fixed!